test/tasks/test_gapic.py

```diff
@@ -158,7 +158,8 @@ def test_execute(self, exec_command, check_output):
             service_yaml='/path/to/service.yaml',
             toolkit_path='/path/to/toolkit',
             aspect='ALL',
-            generator_args='--extra_args'
+            generator_args='--extra_args',
+            proto_package=''
```

Comment:
Would it be possible to add `proto_package` to `generator_args`?

Reply:
What goes inside of generator args? I figure this would best be put in the artman file, because (1) that is where the gapic_yaml option is, and (2) it is an unchanging config value that we shouldn't require the user to write out on the command-line on every invocation.

Reply:
Oh I see.